logging/telemetryConfig_test.go

```diff
@@ -119,3 +119,14 @@ func TestLoadTelemetryConfig(t *testing.T) {
 	require.Equal(t, createTelemetryConfig().Password, tc.Password)
 
 }
+
+func TestLoadTelemetryConfigBlankUsernamePassword(t *testing.T ) {
```

Comment:
you could get some extra credit if you'll add a test for saving the file as well.